modules/fabric-net-firewall/variables.tf

```diff
@@ -16,24 +16,29 @@
 
 variable "network" {
   description = "Name of the network this set of firewall rules applies to."
+  type        = string
 }
 
 variable "project_id" {
   description = "Project id of the project that holds the network."
+  type        = string
 }
 
 variable "internal_ranges_enabled" {
   description = "Create rules for intra-VPC ranges."
+  type        = bool
   default     = false
 }
 
 variable "internal_ranges" {
   description = "IP CIDR ranges for intra-VPC rules."
+  type        = list(string)
   default     = []
 }
 
 variable "internal_allow" {
   description = "Allow rules for internal ranges."
+  type        = list(string)
```

Comment:
This is a list of objects actually.

Reply:
Great! got the fixes in and generated docs.

Reply:
since one parameter is optional, there is no way to work around that with only 1 defined parameter. Removing based off of hashicorp/terraform#19898 . This will not affect the functionality of the module.

Reply:
since one parameter is optional, there is no way to work around that with only 1 defined parameter. Removing based off of hashicorp/terraform#19898 . This will not affect the functionality of the module.